redis/tests/test_async_transaction.rs

```diff
@@ -0,0 +1,63 @@
+use redis::{
+    aio::ConnectionManager, transaction_async, AsyncCommands, Client, Pipeline, RedisError,
+    RedisResult,
+};
+
+const REDIS_URL: &str = "redis://localhost:6379";
+
+#[tokio::test]
+pub async fn test_async_transaction() -> RedisResult<()> {
```

Comment:
please separate into several tests with informative names.

please also test the error paths - specifically, both EXECABORTERROR and errors when running the transaction's commands.

please verify at the end of every test that UNWATCH was called.